lib/VCardConverter.php

```diff
@@ -141,7 +141,10 @@ protected function convertProperty(Component\VCard $input, Component\VCard $outp
                         $newProperty = $output->createProperty('X-ADDRESSBOOKSERVER-KIND', 'GROUP');
                         break;
                 }
+            } elseif ('MEMBER' === $property->name) {
```

Comment:
https://en.wikipedia.org/wiki/VCard#Properties says that MEMBER is a property of vCard V4, But is "Undefined" in V3 and V2. 

Please provide a reference for this being a property of V3.

Reply:
for vCard V2 and V3 - X-ADDRESSBOOKSERVER-MEMBER property is used, but for vCard V4 - MEMBER property is used